src/test/java/org/apache/commons/csv/CSVParserTest.java

```diff
@@ -1281,4 +1283,37 @@ private void validateRecordPosition(final String lineSeparator) throws IOExcepti
 
         parser.close();
     }
+
+    @Test
+    public void parseIssue195andIssue149() throws IOException {
```

Comment:
Please separate into two independent tests, e.g. if the test fails we don't know which JIRA ticket is affected